sigs.yaml

```diff
@@ -101,6 +163,63 @@ sigs:
         description: Design Proposals
       - name: sig-apps-test-failures
         description: Test Failures and Triage
+    subprojects:
+    - name: chart-testing
+      owners:
+      - https://raw.githubusercontent.com/kubernetes-helm/chart-testing/master/OWNERS
+    - name: chartmuseum
+      owners:
+      - https://raw.githubusercontent.com/kubernetes-helm/chartmuseum/master/OWNERS
+    - name: charts
+      meetings:
+      - description: Charts Chat
+        day: Tuesday
+        time: "9:00"
+        tz: "PT (Pacific Time)"
+        frequency: biweekly
+        url: https://zoom.us/j/166909412
+        archive_url: https://docs.google.com/document/d/1h6UTTuNRbFI81higrN3JUV2XxyzqqVjZET4Xz4WTR-8/edit#heading=h.57pbxthvt6k3
+      owners:
+      - https://raw.githubusercontent.com/kubernetes/charts/master/OWNERS
+    - name: charts-tooling
+      owners:
+      - https://raw.githubusercontent.com/kubernetes-helm/charts-tooling/master/OWNERS
+    - name: community
+      owners:
+      - https://raw.githubusercontent.com/kubernetes-helm/community/master/OWNERS
+    - name: examples
+      owners:
+      - https://raw.githubusercontent.com/kubernetes/examples/master/OWNERS
+    - name: helm
+      meetings:
+      - description: Helm Developer call
+        day: Thursday
+        time: "9:30"
+        tz: "PT (Pacific Time)"
+        frequency: weekly
+        url: https://zoom.us/j/4526666954
+        archive_url: https://docs.google.com/document/d/1elWRfvH3AkHdr8pOaqyPbqSZ6ONR-l1Sb9_gapqh8ZA/edit
+      owners:
+      - https://raw.githubusercontent.com/kubernetes/helm/master/OWNERS
+    - name: kompose
+      owners:
+      - https://raw.githubusercontent.com/kubernetes/kompose/master/OWNERS
+    - name: monocular
+      owners:
+      - https://raw.githubusercontent.com/kubernetes-helm/monocular/master/OWNERS
+    - name: rudder-federation
+      owners:
+      - https://raw.githubusercontent.com/kubernetes-helm/rudder-federation/master/OWNERS
+    - name: workloads-api
```

Comment:
jobs and cronjobs should also be here.

Reply:
Done, I updated the description and added the OWNERS files for the controllers and the batch api